fastlane/lanes/release.rb

```diff
@@ -209,7 +209,7 @@ def trigger_buildkite_release_build(branch:, beta:)
     end
   end
 
-  UI.message("Gutenberg version: #{(res.scan(/'([^']*)'/))[0][0]}")
+  UI.message("Gutenberg version: #{res.scan(/'([^']*)'/)[0][0]}")
```

Comment:
This is unrelated to the PR, but a tiny RuboCop violation fix that I though valuable to sneak in.